main.js

```diff
@@ -131,71 +123,22 @@ function changeTimerColor() {
   } else if (currentActivity.category === 'meditate') {
     document.querySelector('.timer-circle-outline').classList.add('meditate-color');
   }
-}
-
-startTimerButton.onclick = startTimer;
+};
 
 function startTimer() {
-  startTimerButton.disabled = true;
-  setInterval(elapseOneSecond, 1000);
-}
-
-function elapseOneSecond() {
-  currentActivity.countdown()
-}
+  setInterval(function () {
+    if (currentActivity.completed === false) {
+      currentActivity.countdown()
+    } else {
+      return;
+    }
+  }, 1000)
+};
 
```

Comment:
Great teamwork getting this countdown to actually stop! I learned more about setInterval during that session than I did struggling with it for 4+ hours.

Reply:
Agreed, nice work everyone for wrestling with this timer! From here, we can invoke a new function above line 133 to change the timer display to say COMPLETED once the countdown completes and also provide the SAVE ACTIVITY button. This button will push currentActivity to localStorage which we can reference to display the activity log.